test/cpp/peer_audio_manager_test.cpp

```
@@ -0,0 +1,185 @@
+const uint8_t kFakeOpus[1] = {0xAB};
+const int kFakeOpusLen = 1;
+const std::string kMacA = "AA:BB:CC:DD:EE:FF";
+const std::string kMacB = "11:22:33:44:55:66";
+
+}  // namespace
+
+// Verify that onVoiceFramePushed returns false for an unregistered peer —
+// the registry lookup guard at the top of the method must fire before
+// any jitter-buffer access.
+void testUnregisteredPeerReturnsFalse() {
+    PeerAudioManager mgr;
```

Clang (14.0.6)

[warning] variable 'mgr' is not initialized

(cppcoreguidelines-init-variables)